### ISSUE TITLE
mxnet r: how to initialize new training session with results from previous session

### ISSUE THREAD

**Issue by Prometheus77** [dmlc/mxnet#6056]
Dumb noob question, but after reading through tutorials and documentation and trying for several hours, I can't figure out how to initialize weights to the results of a previous training session. I'd like to be able to check every 50 rounds or so how the model is fitting my test set, and update learning rate, momentum, etc. accordingly. I've tried setting mx.model.FeedForward.create(..., arg.params=old_model$arg.params) but it doesn't seem to work.

Thanks in advance!

**Comment by Prometheus77** [comment]
I'm closing my own issue. I got it to work in the following way:
model <- mx.model.FeedForward.create(...)
layer_weights <- model$arg.params
model <- mx.model.FeedForward.create(..., arg.params=layer_weights)
